tests/jsx/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -4097,6 +4097,8 @@ f?.(<div/>);
 (<div/>)();
 (<div/>)?.();
 
+new Foo(<Component/>);
```

Comment:
Could you add a test checking that `new Foo((<Component/>))` is turned into `new Foo(<Component/>)` as well?